app/src/main/res/values/strings.xml

```diff
@@ -91,19 +91,19 @@
     <string name="app_settings_notification_settings">Notification settings</string>
     <string name="app_settings_notification_settings_summary">Manage notification channels and their settings</string>
     <string name="app_settings_reset_hints">Reset hints</string>
-    <string name="app_settings_reset_hints_summary">Re-enables hints which have been dismissed previously</string>
+    <string name="app_settings_reset_hints_summary">Reactivates dismissed hints</string>
     <string name="app_settings_reset_hints_success">All hints will be shown again</string>
     <string name="app_settings_sync">Sync</string>
     <string name="app_settings_prefer_tasksorg">Prefer Tasks.org tasks provider</string>
     <string name="app_settings_prefer_tasksorg_on">Will sync tasks with Tasks.org if available</string>
     <string name="app_settings_prefer_tasksorg_off">Will sync tasks with OpenTasks if available</string>
-    <string name="app_settings_prefer_tasksorg_snack">You may need to remove and re-add your account for these changes to take effect.</string>
+    <string name="app_settings_prefer_tasksorg_snack">Remove and re-add your account if you don't see the changes right away.</string>
     <string name="app_settings_connection">Connection</string>
     <string name="app_settings_override_proxy">Override proxy settings</string>
     <string name="app_settings_override_proxy_on">Use custom proxy settings</string>
     <string name="app_settings_override_proxy_off">Use system default proxy settings</string>
-    <string name="app_settings_override_proxy_host">HTTP proxy host name</string>
-    <string name="app_settings_override_proxy_port">HTTP proxy port</string>
+    <string name="app_settings_override_proxy_host">HTTPS proxy host name</string>
+    <string name="app_settings_override_proxy_port">HTTPS proxy port</string>
```

Comment:
We also support plain HTTP proxies. I think the current wording is more clear on that, but happy to change to HTTP(s) or something if you think it's clearer.

Reply:
Good call